examples/index.html

```diff
@@ -9,7 +9,7 @@
   <h1>Web Components</h1>
 
   <ul>
-    <li><a href="/degu-image.html">Degu Image</a>
+    <li><a href="/lit-components.html">Lit Components</a>
```

Comment:
Make the example filenames consistent with the actual filenames, so maybe make this `lit.html` to be consistent with `src/lit` ?

Reply:
Done